data/types.txt

```diff
@@ -1199,3 +1199,4 @@ tourism|theme_park
 tourism|wilderness_hut
 sponsored|cian
 sponsored|holiday
+tourism|resort
```

Comment:
put this under tourism|wilderness_hut, please

Reply:
What do you mean? This file is auto-generated. We cannot reorder types, since that would break all older mwm files.

Reply:
Are both files in this PR auto-generated?

Reply:
No. I added `leisure|resort` to the `mapcss-mapping.csv` and run `generate_drules.sh` to update `types.txt`. We can add new types only to the end of the types list, since there are no holes to fill.

Reply:
Ok. Are there any other "tourism-" types that we can add right now?